// angular-file-uploader/esm2015/lib/angular-file-uploader.module.js
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { AngularFileUploaderComponent } from './angular-file-uploader.component';
import { HttpClientModule } from '@angular/common/http';
import * as ɵngcc0 from '@angular/core';
export class AngularFileUploaderModule {
}
AngularFileUploaderModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: AngularFileUploaderModule });
AngularFileUploaderModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function AngularFileUploaderModule_Factory(t) { return new (t || AngularFileUploaderModule)(); }, imports: [[
            CommonModule,
            HttpClientModule,
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(AngularFileUploaderModule, { declarations: function () { return [AngularFileUploaderComponent]; }, imports: function () { return [CommonModule,
        HttpClientModule]; }, exports: function () { return [AngularFileUploaderComponent]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(AngularFileUploaderModule, [{
        type: NgModule,
        args: [{
                imports: [
                    CommonModule,
                    HttpClientModule,
                ],
                declarations: [AngularFileUploaderComponent],
                exports: [AngularFileUploaderComponent]
            }]
    }], null, null); })();
